src/ui/controllers/suggestion.ts

```diff
@@ -32,6 +32,14 @@ export async function addToPlan(ireq: express.Request, res: express.Response) {
 	}
 	try {
 		await xapi.record(req, course, xapi.Verb.Liked)
+
+		const user = req.user as model.User
+		const record = await learnerRecord.getRecord(user, course) // check record for archived state
+		// need someway to indicate 'unarchived' state retrieved seems to be closest verb
+		if ((record as learnerRecord.CourseRecord).state === 'ARCHIVED')
+			console.log('seting unarchived')
```

Comment:
We shouldn't need to do this - setting 'Liked' should have the same effect as setting unarchived. If this isn't the case then learner-record should be updated so that it does.